components/Favorites/utils.js

```diff
@@ -0,0 +1,200 @@
+export const colors = {
+  RED: 'Красное',
+  WHITE: 'Белое',
+  ROSE: 'Розовое',
+}
+export const sugar = {
+  SWEET: 'Сладкое',
+  DRY: 'Сухое',
+  MEDIUM_DRY: 'Полусухое',
+  MEDIUM: 'Полусладкое',
+}
+export const stores = {
+  'winelab.ru': 'WineLab',
+  'perekrestok.ru': 'Перекрёсток',
+  'https://spb.winestyle.ru': 'WineStyle',
+  'https://www.lenta.com': 'Лента',
+  'amwine.com': 'Ароматный мир',
+  'https://simplewine.ru': 'SimpleWine',
+}
+export const sortingButtons = [
+  {
+    value: 'recommendations',
+    textLabel: 'Рекомендованные',
+  },
+  {
+    value: 'priceAsc',
+    textLabel: 'Возрастанию цен',
+  },
+  {
+    value: 'priceDesc',
+    textLabel: 'Убыванию цен',
+  },
+  {
+    value: 'popularity',
+    textLabel: 'Популярности',
+  },
+]
+export const sortDesc = wines => {
+  const newWines = JSON.parse(JSON.stringify(wines))
+  return newWines.sort((wine1, wine2) => {
+    if (wine1.price) {
+      if (wine2.price) {
+        return wine1.price < wine2.price ? 1 : -1
+      }
+      return wine1.price < wine2.actual_price ? 1 : -1
+    }
+    if (wine2.price) {
+      return wine1.actual_price < wine2.price ? 1 : -1
+    }
+    if (wine1.actual_price === wine2.actual_price) {
+      return 0
+    }
+    return wine1.actual_price < wine2.actual_price ? 1 : -1
+  })
+}
+export const sortAsc = wines => {
+  const newWines = JSON.parse(JSON.stringify(wines))
+  return newWines.sort((wine1, wine2) => {
+    if (wine1.price) {
+      if (wine2.price) {
+        return wine1.price > wine2.price ? 1 : -1
+      }
+      return wine1.price > wine2.actual_price ? 1 : -1
+    }
+    if (wine2.price) {
+      return wine1.actual_price > wine2.price ? 1 : -1
+    }
+    if (wine1.actual_price === wine2.actual_price) {
+      return 0
+    }
+    return wine1.actual_price > wine2.actual_price ? 1 : -1
+  })
+}
+
+export const countries = regions => {
```

Comment:
I think this could be replaced with an object Countries that will receive a country name like this
```javascript
   const COUNTRIES = {
       'Италия': { code 'it', name: 'Италия' }, 
       ....
   }
```